pyeudiw/storage/mongo_storage.py

```diff
@@ -124,6 +127,19 @@ def init_session(self, document_id: str, session_id: str, state: str) -> str:
         self.sessions.insert_one(entity)
 
         return document_id
+    
+    def set_session_retention_ttl(self, ttl: int | None) -> None:
+        self._connect()
+
+        if ttl == None:
```

Comment:
None can be expressed as a non existent or False, and we can have it as an int valued to 0 (zero) as well